Functii/src/c/InputDataValidation.java

```diff
@@ -0,0 +1,26 @@
+package c;
+
+import java.util.Scanner;
+
+public class InputDataValidation {
+
+    public static void main(String[] args) {
+        System.out.println("What time is it?");
+        System.out.println("Please introduce the hour");
+        Scanner sc=new Scanner(System.in);
+        int hour=sc.nextInt();
+        System.out.println("Please introduce the minutes");
+        int min=sc.nextInt();
+        validateData(hour,min);
+    }
+
+    public static void validateData(int hour, int minutes){
```

Comment:
Functia ar fi trebuit sa returneze `boolean`, `true` daca datele introduse sunt valide si `false` daca datele introduse nu sunt valide. De asemenea, verificarea ca functia a returnat `true` sau `false` ar fi trebuit facuta in metoda `main`